applications/eigen_solvers_application/CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+# 
+#   KRATOS _______
+#         / ____(_)___ ____  ____
+#        / __/ / / __ `/ _ \/ __ \
+#       / /___/ / /_/ /  __/ / / /
+#      /_____/_/\__, /\___/_/ /_/ SolversApplication
+#              /____/
+# 
+#   Author: Thomas Oberbichler
+# 
+
+set( CMAKE_INCLUDE_CURRENT_DIR ON )
+
+message( "**** configuring KratosEigenSolversApplication ****" )
+
+if( NOT DEFINED EIGEN_ROOT )
+	message( FATAL_ERROR "EIGEN_ROOT not defined" )
+endif()
+
+if( MSVC )
+	set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /Ox" )
+else()
+	set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O3 -m64 -L$ENV{MKLROOT}/lib/intel64 -Wl,--no-as-needed -lpthread -lm -ldl" )
+endif()
+
+add_definitions( -DNDEBUG -DEIGEN_DEFAULT_TO_ROW_MAJOR -DEIGEN_MPL2_ONLY )
```

Comment:
Is this coupled with the BuildMode `FullDebug`?
I think with full debug you should either add or remove the DNDEBUG

Reply:
It was just to ensure that the optimization is turned on. If the flags are correctly set by Release/FullDebug I can remove it.